### PR TITLE
Add "Merge Responsibilities" document

### PR DESCRIPTION
**Pull request by goldenapples**
Adds a document sketching out the responsibilities of a merger/deployer.
Mostly a skeleton to be fleshed out as we develop the outlines of our
auto-deploy process.

Also adds a few extra points to the code review document which came up
during discussion of merge responsibilities, and one more suggested
reading citation that had some good & relevant points.

### CODE REVIEW

**Comment by goldenapples** [comment]
@danielbachhuber want to give this a review? Also like to get feedback from @davisshaver if you have a chance...

File: team-culture/merge-responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+on Github and wordpress.org/plugins, in the case of WordPress plugins).
+
+## Responsibilities of a code merger
+
+At Fusion, we merge to our primary theme an average of 10-15 times a day.
+Each of these pull requests represents a feature which is important to
+some portion of our stakeholders or our audience. Each also represents
+a potential breakage which can cause disruption, inconvenience, or
+embarrassment. It is the responsibility of the merger to identify these
+possibilities and be prepared to deal with them if they arise.
```
Comment (danielbachhuber) [inline team-culture/merge-responsibilities.md]: We should have some explicit mention here that you should be available for some number of hours after code has gone live.

**Comment by danielbachhuber** [comment]
I think it looks pretty good. @davisshaver ?

Oh, can you change the formatting of the paragraphs so it spans full width? The rest of our documentation isn't constrained to 80 characters.

**Comment by davisshaver** [comment]
@danielbachhuber @goldenapples This reads great. I took care of updating the line widths and adding "stick around following release" lines.

**Comment by goldenapples** [comment]
Thanks! :floppy_disk:

**Pull request merged**